alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -143,6 +145,66 @@ private function appendUiConfToContent($content)
 
 		return $content;
 	}
+
+	private function appendEnvConfigToContent($content)
+	{
+
+		$protocol = infraRequestUtils::getProtocol();
+
+		// The default Kaltura service url:
+		$serviceUrl = requestUtils::getApiCdnHost().'/api_v3';
+		// Default Kaltura CDN url:
+		$cdnUrl = requestUtils::getCdnHost($protocol);
+		// Default Stats URL
+		$statsServiceUrl = ($protocol == "https") ? $this->buildUrl($protocol,"stats_host_https") : $this->buildUrl($protocol,"stats_host");
```

Comment:
BTWits probably worth to move this logic of if-else to the build urL function instead of repeating it multiple times.